### ISSUE TITLE
How Debug Oncall?

### ISSUE THREAD

**Issue by Shuimo03** [grafana/oncall#1421]
Hi, thank you for opening an issue!

Here is a quick checklist:

- [ ] Is it about Cloud or Open Source OnCall?
- [ ] Add OnCall backend & frontend versions.
- [ ] Include labels starting with "part:". Like `part:alertflow` or `part:schedules`.
- [ ] Include labels like `bug` or `feature request`.
- [ ] If it's a bug, include logs.

Hi, I want  Add feature that webhook on user Info tab, but I dont know debug Oncall

**Comment by Konstantinov-Innokentii** [comment]
Hello, @Shuimo03. I'm sure you will find this [readme](https://github.com/grafana/oncall/blob/dev/dev/README.md) useful. If you are running oncall engine inside docker it's possible to confugire your IDE for debug ([Pycharm example](https://www.jetbrains.com/help/pycharm/using-docker-as-a-remote-interpreter.html#debug)). 
I personally start oncall engine and celery on my machine, via manage.py commands, while running database and other services in docker.

I know, that my colleague @mderynck works on improving webhooks, make sure to discuss with him your feature in order to not to duplicate work.

**Comment by Shuimo03** [comment]
Yes, thank you very much for your suggestion, but I was thinking of adding a webhook to the personal page in the user page so that alerts can be sent to a third party communication tool to find the corresponding person
